project/AppDependencies.scala

```diff
@@ -4,28 +4,35 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"                %% "bootstrap-backend-play-28"   % "5.19.0",
+    "uk.gov.hmrc"                %% "bootstrap-backend-play-28"   % "7.13.0",
     "uk.gov.hmrc.mongo"          %% "hmrc-mongo-play-28"          % "0.68.0",
     "com.enragedginger"          %% "akka-quartz-scheduler"       % "1.9.1-akka-2.6.x",
     "org.typelevel"              %% "cats-effect"                 % "2.2.0-RC3",
-    "org.apache.commons"          % "commons-text"                % "1.9",
+    "org.apache.commons"          % "commons-text"                % "1.10.0",
+    "org.apache.commons"          % "commons-io"                % "1.3.2",
     "org.xhtmlrenderer"           % "flying-saucer-pdf"           % "9.1.20",
     "org.apache.poi"              % "poi-ooxml"                   % "4.1.2",
     "org.julienrf"               %% "play-json-derived-codecs"    % "10.0.2",
     "com.chuusai"                %% "shapeless"                   % "2.3.3",
-    "uk.gov.hmrc.objectstore"    %% "object-store-client-play-28" % "0.39.0",
+    "uk.gov.hmrc.objectstore"    %% "object-store-client-play-28" % "1.0.0",
   )
 
   val test = Seq(
-    "uk.gov.hmrc"            %% "bootstrap-test-play-28"    % "5.19.0"  % Test,
+    "uk.gov.hmrc"            %% "bootstrap-test-play-28"    % "7.13.0"  % Test,
     "org.scalatest"          %% "scalatest"                 % "3.1.4"   % Test,
     "org.scalacheck"         %% "scalacheck"                % "1.14.3"  % Test,
     "org.scalatestplus"      %% "scalacheck-1-14"           % "3.1.0.0" % Test,
     "de.flapdoodle.embed"     % "de.flapdoodle.embed.mongo" % "3.5.3"   % Test,
     "com.typesafe.play"      %% "play-test"                 % current   % Test,
     "org.mockito"            %% "mockito-scala"             % "1.16.42" % Test,
     "org.mockito"            %% "mockito-scala-scalatest"   % "1.16.23" % Test,
-    "com.typesafe.akka"      %% "akka-testkit"              % "2.6.17"  % Test,
+    "com.typesafe.akka"      %% "akka-testkit"              % "2.6.20"  % Test,
+    "com.typesafe.akka"      %% "akka-slf4j"                % "2.6.20" % Test,
+    "com.typesafe.akka"      %% "akka-protobuf-v3"          % "2.6.20" % Test,
+    "com.typesafe.akka"      %% "akka-serialization-jackson"% "2.6.20" % Test,
+    "com.typesafe.akka"      %% "akka-stream"               % "2.6.20" % Test,
+    "com.typesafe.akka"      %% "akka-actor-typed"          % "2.6.20" % Test,
+    "com.typesafe.akka"      %% "akka-actor"                % "2.6.20" % Test,
```

Comment:
Are these dependencies necessary?

Reply:
Yes, some tests are failing without these

Reply:
What test are failing without these?

Reply:
No you're right, turns out the errors were from somewhere else. I've removed the unnecessary  dependencies in the new commit.